src/components/AccountLists/AccountLists.tsx

```diff
@@ -76,18 +76,37 @@ const AccountLists = ({ data }: Props): ReactElement => {
               ({
                 id,
                 name,
-                monthlyGoal,
+                monthlyGoal: preferencesGoal,
                 receivedPledges,
                 totalPledges,
-                currency,
+                currency: preferencesCurrency,
+                healthIndicatorData,
               }) => {
+                const monthlyGoal =
+                  preferencesGoal ?? healthIndicatorData?.machineCalculatedGoal;
+                const currency =
+                  typeof preferencesGoal === 'number'
+                    ? preferencesCurrency
+                    : healthIndicatorData?.machineCalculatedGoalCurrency;
```

Comment:
Should we make these variables use the same IF statements to kae it easier to read?

I know `monthlyGoal` will not equal `preferencesGoal`  while `currency` equals `healthIndicatorData?.machineCalculatedGoalCurrency`. but it could be easier to understand if it was like the below

```suggestion
                const usePreferencesGoal = typeof preferencesGoal === 'number';
                const monthlyGoal = usePreferencesGoal
                  ? preferencesGoal
                  : healthIndicatorData?.machineCalculatedGoal;
                const currency = usePreferencesGoal
                    ? preferencesCurrency
                    : healthIndicatorData?.machineCalculatedGoalCurrency;
```